extension/utils.go

```diff
@@ -0,0 +1,22 @@
+package extension
+
+import (
+	"os"
+)
+
+// GetS3ExtensionBucketHost returns the bucket to use for accessing the crx files
+func GetS3ExtensionBucketHost() string {
+	s3BucketHost, ok := os.LookupEnv("S3_EXTENSIONS_BUCKET_HOST")
+	if !ok {
+		s3BucketHost = "brave-core-ext.s3.brave.com"
+	}
+	return s3BucketHost
+}
+
+// UpdateStatus returns the status of an update response for an extension
+func UpdateStatus(extension Extension) string {
```

Comment:
This is a really minor nit, but at first glance I was looking for an assignment operator because the function sounds like it's updating a status. Maybe `GetUpdateStatus` would be a better name?